### DISCUSSION TITLE
Cloud Accounts and Commonhaus foundation?

### DISCUSSION THREAD

**Discussion by sdelamo** [commonhaus/foundation#166] [Q&A]
The infrastructure of an open-source project often requires Cloud Accounts (AWS, Google Cloud, Azure, Oracle Cloud) to host services. Cloud vendors often provide free credits for open source, but those accounts require an association with a credit card. 

How will it work in the Commonhaus Foundation? Will the Commonhaus Foundation create those cloud accounts, associate them with the Commonhaus tax identity and Commonhaus credit card, and then grant access to the open-source project managers?

**Comment by ebullient** [comment]
In general, as a fiscal host, that is how things should work, and the foundation does have accounts to do that. The actual mechanics are to be determined, as we're still in the bootstrapping phase.

**Comment by marc0der** [comment]
SDKMAN is heavily reliant on cloud infrastructure for it's microservices backend. Today, we fund this out of the community contributions that are made each month, and I don't expect that to change.

The only difference that I'm aware of is that CH will become our fiscal host instead of OC, and CH will scoop a small percentage of our project's contributions to run CH. @ebullient can keep me honest here 😄 

At the end of the month, I'd still pay cloud infrastructure with my own money, then claim back and even approve it like I currently do. The CH fiscaal host will then reimburse me. Again, @ebullient, please correct me if I've misunderstood.
